package.json

```diff
@@ -394,6 +394,11 @@
           "type": "boolean",
           "description": "%java.debugger.configuration.enableRunDebugCodeLens.description%",
           "default": true
+        },
+        "java.debug.settings.forceBuildBeforeLaunch": {
+          "type": "boolean",
+          "description": "%java.debugger.configuration.forceBuildBeforeLaunch%",
```

Comment:
Just curiously about the nls file. Could we group them together under some specified folder but not under root. It will be a disaster if we support more languages in future.